framework/messages/de/yii.php

```diff
@@ -26,10 +26,10 @@
     ' and ' => ' und ',
     '(not set)' => '(nicht gesetzt)',
     'An internal server error occurred.' => 'Es ist ein interner Serverfehler aufgetreten.',
-    'Are you sure you want to delete this item?' => 'Wollen Sie diesen Eintrag wirklich löschen?',
+    'Are you sure you want to delete this item?' => 'Wollen Sie diesen Artikel wirklich löschen?',
```

Comment:
Is this for deleting records from the db? Actually I'd say "Eintrag" is more generic, but debatable. @cebe

Reply:
Should be "Eintrag", which is more generic than "Artikel".